src/modules/Channel/components/MessageList/hooks/useSetScrollToBottom.ts

```diff
@@ -20,13 +16,15 @@ export function useSetScrollToBottom({
   const scrollCb = (e: React.UIEvent<HTMLDivElement, UIEvent>) => {
     const element = e.target as HTMLDivElement;
     try {
-      setScrollBottom(element.scrollHeight - element.scrollTop - element.offsetHeight);
+      setScrollBottom(
+        element.scrollHeight - element.scrollTop - element.offsetHeight,
+      );
     } catch {
       //
     }
   };
   return {
     scrollBottom,
-    scrollToBottomHandler: useDebounce(scrollCb, DELAY),
+    scrollToBottomHandler: useThrottleCallback(scrollCb, DELAY, { trailing: true }),
```

Comment:
This has caused a slow update of the scrollBottom value.
The scrollBottom value is used in a place where adjustments are made when the size of the content within the current scroll view changes, and as a result, it was not being updated with the latest values.
(e.g., When replying, the scroll does not go to the bottom.)